auto_joiner.py

```diff
@@ -304,6 +317,17 @@ def main():
         if use_web_instead is not None:
             use_web_instead.click()
 
+    # if u have additioan account some something link https://imgur.com/a/CiifUzF    following will excecute
+    if config['additional_acc']  > 1 :
+        select_change_acc=wait_until_found_xpath("/html/body/div[1]/div[1]/app-header-bar/div/div/button[1]",20)
+        if select_change_acc is not None :
+            select_change_acc.click()
+            time.sleep(1)
+            change_acc = wait_until_found_xpath(f"/html/body/div[5]/div/ul/settings-dropdown-tenants/li[{config['additional_acc']}]/button[1]",20)
+            # /html/body/div[5]/div/ul/settings-dropdown-tenants/li[2] also usable
+            if change_acc is not None :
+                change_acc.click()
+    
```

Comment:
Change from xpath to css selector so it is the same as in the other parts of the code, change phrasing and make more error proof
```suggestion
    # if additional organisations are setup in the config file
    if config['organisation_num']  > 1 :
        additional_org_num = config['organisation_num']
        select_change_org=wait_until_found("button.tenant-switcher",20)
        if select_change_org is not None :
            select_change_org.click()
            
            change_org = wait_until_found(f"li.tenant-option[aria-posinset='{additional_org_num}']",20)
            if change_org is not None:
                change_org.click()
    
```